test/salt/pillar/postgres.sls

```diff
@@ -1,7 +1,7 @@
 # Port to use for the cluster -- can be used to provide a non-standard port
 # NOTE: If already set in the minion config, that value takes priority
 
-{%- if not (grains.os_family == 'Debian' or grains.osfinger == 'Leap-15') %}
+{%- if not (grains.os_family == 'Debian' or salt['grains.get']('osfinger') == 'Leap-15' %}
```

Comment:
Here the parentheses were valid. But we could remove them by simplifying Boolean logic:
`not (var == val or var == val)` is the same as `var != val or var != val`.

Reply:
No problem - that's done with updated logic.